docs/containers/docker-compose.md

```diff
@@ -193,6 +193,76 @@ volumes:
     - /host-folder-path:/container-folder-path
 ```
 
+## Docker Compose with multiple Compose files
+Workspaces can have multiple docker-compose files to handle different environments like development, test and production. The content of the configuration can be split into multiple files. For example, a base compose file that defines the common information for all environments and override files that defines environment specific information. When these files are passes as input to `docker-compose` command, it will combine these files into single configuration. By default the **Docker: Compose Up** command will pass a single file as input to compose command. But the compose command can be customized to pass in multiple files using [command customization](https://code.visualstudio.com/docs/containers/reference#_command-customization). This can be also be achieved using [custom task](https://code.visualstudio.com/docs/editor/tasks#_custom-tasks) to invokes the `docker-compose` command with desired parameters.
```

Comment:
"passes" should be "passed".
"into single" should be "into a single"
"to compose command" should be "to the compose command"
"This can be also be achieved" should be "this can also be acheived"